### ISSUE TITLE
Chrome.

### ISSUE THREAD

**Issue by Alex313031** [egrath/chromeos-image-downloader#1]
Hey, I didn't know where else to contact you. I'm a Chromium dev and make my own Chromium fork called Thorium https://github.com/Alex313031/Thorium and builds of ChromiumOS https://github.com/Alex313031/ChromiumOS I had been looking for a way to download the ChromeOS wallpapers, and came across your script. 
How did you figure out how to implement it? And can you update it? Or at least tell me how to? There are more categories now, especially the material and radiance ones. For example there's "city": "\x0a\x04\x63\x69\x74\x79\x12\x05\x65\x6e\x2d\x55\x53", How would you or I update this for these new categories, like how do you know what numbers to use?

**Comment by egrath** [comment]
Hi, i will look into it. Essentially i just use Googles Protocol Buffer Protocol to do the heavy lifting.

**Comment by egrath** [comment]
Just looked into it, checked the corresponding commit logs, updated to the latest protocol buffer descriptions and verified the URLs. As you mentioned that there are new categories, i fired up my Chromebox but the categories available in the downloader and directly in Chrome OS are the same:

![Screenshot_select-area_20220422072920](https://user-images.githubusercontent.com/1955895/164609912-a888daa0-fc27-466f-9fc4-6244eb2fad06.png)

As far as i know, Google delivers different categories according to your geographic location. As it seems that you are US based, i checked that too, but got the same results:

```
./down.py --list-collections --debug --region US
```
 
Can you please provide me with your output of the later command.

**Comment by Alex313031** [comment]
@egrath  Apparently some of the new ones are only activated on CrOS with the "staging wallpaper server" flag. Which when looking in the source code gives this.
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ash/wallpaper_handlers/wallpaper_handlers.cc?q=kTestWallpaperServer

Note the different server. Perhaps you would have to modify the url in the down.py script.

EDIT: Wait I just found you already thought of that.
BTW I was using your nodejs gist of this project, not the main one here. This seems more complete and using --server I was able to download all of them including the new ones.

This is a great project! Would you mind if I integrated it into my ChromiumOS builds? I would provide credit in the main license as well as a license in the .ebuild for the packages I would create that just depends on the pip packages this needs and installs this whole repo in a subdir of /usr/local/bin.

**Comment by Alex313031** [comment]
@egrath Sorry, I only came across the gist. I assumed they were the same. If I had just used this from the beginning it would have been fine. BUT I think it's good I opened an issue anyway, because now you updated the protobuf definition and required package versions. Thank you for bein smart at python, unlike me lol.

I ran it with --server prod, dev, and staging, and then merged the output folders together replacing any preexisting ones to get a single folder with all the wallpapers in one go. Exactly what I wanted!
